packages/shared/src/components/squads/settings/SquadModerationSettingsSection.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { ReactElement, useState } from 'react';
+import { SquadSettingsSection } from './SquadSettingsSection';
+import { Radio } from '../../fields/Radio';
+import { SourceMemberRole } from '../../../graphql/sources';
+import { Switch } from '../../fields/Switch';
+import { WidgetCard } from '../../widgets/WidgetCard';
+import { SimpleTooltip } from '../../tooltips';
+import { useToggle } from '../../../hooks/useToggle';
+
+interface SquadModerationSettingsSectionProps {
+  initialMemberPostingRole?: SourceMemberRole;
+  initialMemberInviteRole?: SourceMemberRole;
+  initialModerationRequired?: boolean;
+}
+
+const memberRoleOptions = [
+  {
+    label: 'All members (recommended)',
+    value: SourceMemberRole.Member,
+  },
+  {
+    label: 'Only Moderators',
```

Comment:
We never do this, expect for product terms (Squad for instance)
```suggestion
    label: 'Only moderators',
```

Reply:
In figma it says "the moderation settings sections include text changes, and new "require post approval" option.", There's multiple texts that don't match with live, so I assumed it referred to all text changes. "Moderators" is capitalized in the design